reverse_proxy.rst

```diff
@@ -1,33 +1,78 @@
-=============
+==============================
+Certificates and reverse proxy
+==============================
+
 Reverse proxy
 =============
 
-.. warning::
-
-   This feature is still under development and does not have a user interface yet. It can currently only be configured through the command line.
-
-NethSecurity provides a reverse proxy using nginx.
-A reverse proxy is a server that sits in front of one or more web servers and forwards requests to them. It can be used to improve performance, security, and reliability.
+NethSecurity provides a reverse proxy using `nginx <https://nginx.org>`_.
+A reverse proxy, sometimes called also proxy pass,  is a server that sits in front of one or more web servers and forwards requests to them.
+It can be used to improve performance, security, and reliability.
 
 In simpler terms, a reverse proxy is like a traffic cop for web servers. It directs incoming requests to the appropriate server and sends back the responses.
 
 Reverse proxies are often used to improve performance by caching static content and distributing traffic across multiple servers. They can also be used to increase security by implementing the TLS endpoint.
 
-See the `developer manual <https://dev.nethsecurity.org/packages/nginx/>`_ for a guide on how to configure it from the command line.
+This page allows users to configure proxy pass settings, specifying whether the rule applies to a domain or a path.
+For domain configurations, users can select a certificate.
+The destination URL determines where incoming requests are forwarded, and the allowed network field provides the option
+to restrict access to specific CIDR-formatted networks. A description can be added for clarity.
+
+To configure a new proxy pass, click on the :guilabel:`Add reverse proxy` and customize the following options:
+
+- ``Type``: choose between Domain or Path.
+  If type is ``Path``, enter the resource path starting with a '/' for matching rules (e.g., ``/resource-path``).
+  If type is ``Domain``, enter the fully qualified domain name for website matching rules. Select also an associated :ref:`certificate <certificates-section>`.
+- ``Destination URL``: specify the forwarding location for incoming requests (e.g., ``http://destination-server:port/path``).
+- ``Allowed networks``: define allowed IPv4/IPv6 networks in CIDR format. By default, accessible from anyone.
+- ``Description``: optionally, add a description for clarity.
+
+Additional information:
+
+- Headers sent to destination server: X-Forwarded-Proto, X-Forwarded-For, X-Real-IP are always sent.
+- Certificate validation: of the destination uses HTTPS, the certificate is not validated to avoid errors on misconfigured servers.
```

Comment:
```suggestion
- Certificate validation: if the destination uses HTTPS, the certificate is not validated to avoid errors on misconfigured servers.
```